server/src/scoring/handlers/userHandlers.ts

```diff
@@ -0,0 +1,27 @@
+﻿import { RequestHandler } from 'express';
+
+import { prisma } from '~/common/prisma';
+
+import { validateUser } from './../validations/userValidate';
+
+export const getUserMandatoryInfo: RequestHandler = async (req, res) => {
+  const { error } = validateUser(req.body);
+
+  if (error) return res.status(400).send(error.details[0]?.message);
+
+  if (!req.user) return res.status(401).send('brak autoryzacji');
+
+  const { contractType, toWorkDistance, nickname } = req.body;
+  const nick = nickname?.length > 0 ? nickname : req.user.nickname;
+  const user = await prisma.user.update({
+    where: { email: req.user.email },
+    data: {
+      nickname: nick,
+      contractType,
+      toWorkDistance,
+      avatar: req.user.avatar,
```

Comment:
Tego raczej nie musisz przekazywać, jeżeli nie ruszysz jakiegoś pola to prisma nie wyczyści wartości w bazie